test/test_autograd.py

```diff
@@ -3333,6 +3333,318 @@ def test_checkpointing(self):
         mean_combined = torch.stack(feat_combined).mean()
         mean_combined.backward()
 
+    def test_autograd_views_codegen(self):
+        # This is not necessarily the absolute correct behavior, but this is the current
+        # one. This test is here to make sure that any change to this behavior is detected
+        # and not silent.
```

Comment:
Maybe, explain what acceptable other behaviors are? I.e. what are the degrees of freedom in the test?

Reply:
Changing any of these test will be a BC-breaking change. All these tests use only public api so users could be using it.